src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/NeoFunnel/NeoFunnel.java

```diff
@@ -0,0 +1,42 @@
+package edu.greenblitz.robotName.subsystems.shooter.Funnel.NeoFunnel;
+
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.FunnelInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.IFunnel;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+import edu.wpi.first.math.filter.Debouncer;
+
+import static edu.greenblitz.robotName.subsystems.shooter.Funnel.NeoFunnel.NeoFunnelConstants.*;
+
+public class NeoFunnel implements IFunnel {
+	
+	private GBSparkMax motor;
+
+	private Debouncer debouncer;
+
+	public NeoFunnel() {
+		motor = new GBSparkMax(FUNNEL_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+		motor.config(FUNNEL_CONFIG_OBJECT);
+		motor.getReverseLimitSwitch(SWITCH_TYPE).enableLimitSwitch(IS_SENSOR_EXIST);
```

Comment:
Why is it a limit switch? We still want the ability to take the object backwards to bring in to the endeffector so i dont think it should be a limit.